bitnami/influxdb/README.md

```diff
@@ -51,169 +51,173 @@ The command removes all the Kubernetes components associated with the chart and
+| `backup.uploadProviders.azure.existingSecret`     | Name of existing secret object                                                                                                                                                                                                                                       | `""`                                                    |
+| `backup.uploadProviders.azure.containerName`      | destination container                                                                                                                                                                                                                                                | `"influxdb-container"`                                  |
+| `backup.uploadProviders.azure.image.registry`     | Azure CLI image registry                                                                                                                                                                                                                                             | `docker.io`                                             |
+| `backup.uploadProviders.azure.image.repository`   | Azure CLI image name                                                                                                                                                                                                                                                 | `bitnami/azure-cli`                                     |
+| `backup.uploadProviders.azure.image.tag`          | Azure CLI image tag                                                                                                                                                                                                                                                  | `{TAG_NAME}`                                            |
+| `backup.uploadProviders.azure.image.pullPolicy`   | Azure CLI image pull policy                                                                                                                                                                                                                                          | `IfNotPresent`                                          |
+| `backup.uploadProviders.azure.image.pullSecrets`  | Specify docker-registry secret names as an array                                                                                                                                                                                                                     | `[]` (does not add image pull secrets to deployed pods) |
```

Comment:
Can we organize the values per component/section as we do in other charts?

Reply:
done!